HW4/src/isBigger.js

```diff
@@ -0,0 +1,6 @@
+function isBigger(a, b){
+    return a > b;
+}
+
+console.log(isBigger(9, 6));
+
```

Comment:
use console.log() in the functions

Reply:
Why?
it doesn’t affect how the code works!